### ISSUE TITLE
Feedback on Tributary Impact Riverscapes Project

### ISSUE THREAD

**Issue by joewheaton** [Riverscapes/TributaryImpact#1]
Hi @banderson1618 
Here's some feedback on the project: https://youtu.be/E0pvlpgolJc

Please implement something similar for layer package and symbology, etc. to what I show in video.

I've packaged that up in a layer package for you: https://usu.box.com/s/skxrikx1wf815pt9bvz4ydha6wm56t89

**Comment by banderson1618** [comment]
Is there any existing code that does this that you know of? I can probably figure it out on my own, but having an example to work off of would be helpful.

**Comment by banderson1618** [comment]
Another question: Would you prefer the output to only be a layer package, or should we still produce a project with the raw shape files, and have the layer package as one of the outputs?

**Comment by banderson1618** [comment]
I've now made it so that the code creates a layer file for each output and provides them with the symbology in the layers in the layer package that you have me. I also removed a "Drainage Area Difference" cutoff that kept us from adding a point to the output if the difference in drainage area was too big between the mainstem and the tributary. I'm running the tool on the Asotin, I'll zip up the output and post it here when it's done.

**Comment by banderson1618** [comment]
Here's a Google Drive link to what the Tributary Impact Tool currently gives:

https://drive.google.com/file/d/1clnB4bmMbLBDlXoyuDb09kS6IULcThMq/view

@joewheaton  @wally-mac, let me know what you think. Do we want to make a layer package, and if so, should that layer package include the inputs?

Cheers,
Braden

**Comment by wally-mac** [comment]
Yes, we'll need to produce a layer package.  Please work with Mic (ma697@nau.edu ) who can help you develop symbology that follows our cartographic standards.

Thanks,
Wally

**Comment by banderson1618** [comment]
There's an error in creating the layer package that's only applicable in ArcGIS 10.6. We ran the tool just fine on Adan's computer, which is running 10.5.

**Comment by banderson1618** [comment]
Outside of that, the tool applies symbology just fine to the layers, both inputs and outputs. We don't have a file structure set up within the layer package, but other than that, we're just about good to go.

**Comment by wally-mac** [comment]
Thanks for the update.  Once you have it completed please put a link to it here and I'll have a look.

**Comment by banderson1618** [comment]
https://drive.google.com/file/d/1M3vK1F0wcC_1RDRhaLCTMOd9QyvK4hCr/view?usp=sharing

Here's a Google Drive link to the latest output of the tool, including a layer package. I don't know how to get a folder structure in the layer package, but other than that it looks pretty good.

**Comment by wally-mac** [comment]
Please ask Mic "how to get a folder structure in the layer package". Then once you have the layer package with the correct folder structure I'll have a look at it.
